Outreachy Dec 2021 to Mar 2022.md

```diff
@@ -0,0 +1,39 @@
+Oppia is excited to be participating in [Outreachy](https://www.outreachy.org/)'s December 2021 to March 2022 round! Outreachy is a diversity initiative that provides paid, remote internships to people subject to systemic bias and impacted by under-representation in the technical industry where they are living.
+
+Thank you for your interest in contributing to our project! Here you can find initial guidance on how to get started with the required tasks listed on the project [Scale Community Outreach and Conduct Research on Oppia's Lessons](https://www.outreachy.org/outreachy-december-2021-internship-round/communities/oppia/#scale-community-outreach-and-conduct-research-on-o).
+
+If you have any questions, feel free to send them to outreachy-questions@oppia.org or send them **directly to mentors by email**.
+
+## Getting Started
+
+Please read through the topics below to get started on the three required tasks for the contribution period:
+
+### User Studies
+
+*  For the user studies, start reading [this guide](https://docs.google.com/document/d/1j1S9HnoesGZMNeYccV8rXZ2bZwPhKV5-B6j9CnPRM_I/edit?usp=sharing)  that provides information on how to properly perform a user study or learner feedback session on Oppia's math lessons. You can also make a copy and use [this checklist](https://docs.google.com/spreadsheets/d/1ybGCKSsa8fIqcBliavShN06f_PakiiBPPxLg6d48R2w/edit?usp=sharing) to help make sure you don't miss any important steps. 
+* We recommend that you conduct learner feedback sessions on at least 3 consecutive chapters, using either the Web platform or the Android App, on any of the math topics inside [Oppia Classroom](https://docs.google.com/document/d/1j1S9HnoesGZMNeYccV8rXZ2bZwPhKV5-B6j9CnPRM_I/edit?usp=sharing). One student is enough for the contributing period, but you can work with more if you want to.
+*  Remember that for the sessions, you will have to make sure that the chapters you will use and the materials (surveys and tests) listed in the guide/checklist are translated in the student's language. Also remember that the report on your findings is your main deliverable for this task, and we recommend you follow the report template listed on the guide/checklist to save effort and avoid missing relevant information. 
+
+### Reaching and defining pilot plans with NGOs
+
+* For reaching and drafting pilot plans with NGOs, we recommend that you start finding and listing a few NGOs related to children's education that you could potentially work with to use Oppia lessons. You will need to contact and meet with the NGOs in order to introduce Oppia and talk about the NGO activities. If you need guidance on how to strategically partner with organizations by taking a programmatic approach, please consult [Oppia Menu of Services](https://docs.google.com/document/d/1sC7lfTQ_iT6dx78wHjL1S-ldJDSUC3UEltnB0j0yexA/edit?usp=sharing ) 
```

Comment:
Do you mean "reaching _out_ and defining pilot plans"? I'm not sure what you were intending here, so feel free to leave it as is if the current text is what you meant!

Reply:
Yes, that's what I mean. I just added "out"